### DISCUSSION TITLE
How to use picsimlab_tool from platformio ?

### DISCUSSION THREAD

**Discussion by dattodroid** [lcgamboa/picsimlab#119] [Q&A] [answered]
Does someone have an example  on how to use  picsimlab_tool from platformio ?
Thanks.

**Comment by lcgamboa** [comment, marked as answer]
Yes. There are examples in the documentation of version 0.9.2. 
Links for:
- [AVR](https://lcgamboa.github.io/picsimlab_docs/0.9.2/avrgdbDebug.html#x47-460006.2.1)
- [STM32](https://lcgamboa.github.io/picsimlab_docs/0.9.2/armgdbDebug.html#x50-490006.3.1)
- [ESP32](https://lcgamboa.github.io/picsimlab_docs/0.9.2/ESP32gdbDebug.html#x52-510006.4.1)

**Reply by dattodroid** [reply]
Thank you.
